src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -660,7 +660,7 @@ void TableDockWidget::exportGroupsToSpreadsheet() {
                               return group.getCompound()->type() == Compound::Type::PRM;
                             });
   bool prmGroupExists = prmGroupAt != end(allgroups);
-  bool includeSetNamesLines = true;
+  bool includeSetNamesLines = false; // when should this be true?
```

Comment:
This should be true when the user selects the "Group Summary Matrix Format with Set Names" option.

Reply:
Let me know if that option is working.

Reply:
I see. Thanks for the info. I'll make sure it works.